test-harness/specs/E2E-Accept-Content-Negotiation-AC-1.oas2.txt

```diff
@@ -22,7 +22,7 @@ mock -p 4010
 curl -i -X GET http://localhost:4010/todos -H "accept: application/json"
 ====expect====
 HTTP/1.1 200 OK
-content-type: application/json
+content-type: application/json; charset=utf-8
```

Comment:
This should not be modified. The OAS document does not specify the charset, and so the response should not.

Reply:
Updating https://github.com/stoplightio/prism/pull/620#discussion_r325693751 resulted in removing all `; charset=utf-8` from tests. Looks like an unexpected fix.